src/main/java/com/spotify/futures/CompletableFutures.java

```diff
@@ -360,6 +359,124 @@ public static void checkCompleted(CompletionStage<?> stage) {
   }
 
   /**
+   * Composes multiple stages into another stage using a function.
+   *
+   * @param a        the first stage.
+   * @param b        the second stage.
+   * @param function the composing function.
+   * @param <R>      the type of the composed {@link CompletionStage}.
+   * @param <A>      the type of the first stage's value.
+   * @param <B>      the type of the second stage's value.
+   * @return a stage that is composed from the input stages using the function.
```

Comment:
Perhaps copy+modify the documentation from `CompletionStage.toCompletableFuture` stating:
```
@throw UnsupportedOperationException - if this implementation does not interoperate with CompletableFuture
```